spec/mutations/steps/create_spec.rb

```diff
@@ -3,7 +3,7 @@
 describe Steps::Create do
   let(:user) { FactoryGirl.create(:user) }
   let(:mutation) { Steps::Create }
-  let(:valid_params) { {message_type: 'move_rel',
+  let(:valid_params) { {message_type: 'move_relative',
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.<br>Space inside { missing.